package/rmservewacominput/package

```diff
@@ -5,16 +5,24 @@
 pkgnames=(rmservewacominput)
 pkgdesc="Serve pen input on port 33333"
 url=https://github.com/LinusCDE/rmWacomToMouse
-pkgver=0.2.5-9
-timestamp=2020-09-06T23:23Z
+pkgver=0.3.0-1
+timestamp=2021-03-10T18:36Z
 section="utils"
 maintainer="Linus K. <linus@cosmos-ink.net>"
 license=MIT
 depends=(appmarkable)
 
-image=base:v1.3.2
-source=(https://github.com/LinusCDE/rmWacomToMouse/archive/1a45d3446c8bc11de04800a702576056016875ed.zip)
-sha256sums=(4d51ff9d6b8f57fe4970b4a0adebafe3513ddeb521a2707b81a42d32f73dadb1)
+image=base:v1.4
+source=(
+    https://github.com/LinusCDE/rmWacomToMouse/archive/1a45d3446c8bc11de04800a702576056016875ed.zip
+    rmservewacominput.draft
+    rmservewacominput_gui
+)
+sha256sums=(
+    4d51ff9d6b8f57fe4970b4a0adebafe3513ddeb521a2707b81a42d32f73dadb1
+    SKIP
+    SKIP
+)
```

Comment:
This should probably be changed to the latest commit so that the device detection code is included.

```suggestion
source=(
    https://github.com/LinusCDE/rmWacomToMouse/archive/fd1c5454b65f456f6e890b99109e50a8f576dad1.zip
    rmservewacominput.draft
    rmservewacominput_gui
)
sha256sums=(
    c2c15b519a8352a5b52e1b15154e4024acc423552c6cf7559c23fcb8beffcd56
    SKIP
    SKIP
)
```

Reply:
Yeah. Somehow missed that. Good catch!